cli/formicactl.go

```diff
@@ -49,12 +52,16 @@ var (
 			newController = controller.NewController(newControllerConfig)
 
 			newFileSystem = filesystemreal.NewFileSystem()
+
+			newTaskConfig := task.DefaultTaskConfig()
+			newTask = task.NewTask(newTaskConfig)
```

Comment:
I suggest having a general look at the PR and then asking questions. Maybe naming needs to be fixed here in some places. A "task" here refers to the task managing functionality, not a task per se.